app/controllers/think_feel_do_dashboard/moderators_controller.rb

```diff
@@ -4,7 +4,7 @@ module ThinkFeelDoDashboard
   # Allows Coaches/Clinicians to moderate.
   # That is, log in as a participant
   class ModeratorsController < ApplicationController
-    before_action :authenticate_user!, :set_group
```

Comment:
I am writing specs for this, but it `authenticate_user!` is in `ApplicationController`